media/css/quantum/_common.scss

```diff
@@ -0,0 +1,165 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../pebbles/includes/lib';
+@import 'lib';
+
+
+$color-quantum-blue: #0a84ff;
+
+/* -------------------------------------------------------------------------- */
+// Common elements
+
+body {
+    @include open-sans;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-weight: normal;
+}
+
+/* -------------------------------------------------------------------------- */
+// Key features sections
+
+.key-features-section {
+    background: #fff;
+
+    h2 {
+        @include font-size-level3;
+        color: $color-quantum-blue;
+        font-weight: bold;
```

Comment:
On the `/firefox/quantum/` page design, these titles we're previously all in regular. This change leaves a bit of an odd mix of weights on the page?

Reply:
Will remove `bold` for consistency.